### PR TITLE
Content level 11

### PR DESCRIPTION
**Pull request by MarleenGilsing**
Content for level 11

**Description**

Content for level 11 in EN and NL for the adventures: story, songs, caculator, fortune, restaurant and the new adventure piggybank.
In order for the songs and fortune adventures to work, Issue #1231 should be fixed.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

### CODE REVIEW

File: coursedata/adventures/en.yaml [hunk 1]
```diff
@@ -1528,6 +1602,28 @@ adventures:
...
+                    In this adventure you learn how to make a digital piggy bank, to calculate how much money you have and how long you need to save up to buy what you want!
+
+                    ## Example Hedy Code
+                    ```
+                    print 'The digital piggy bank'
+                    wish is ask 'What would you like to buy?'
+                    price is ask 'How much does that cost?'
+                    saved is ask 'How much money have you saved already?'
+                    allowance is ask 'How much pocket money do you get per week?'
+                    tosave is price - saved
```
Comment (Felienne) [inline coursedata/adventures/en.yaml:1623]: @MarleenGilsing Ik zou het mooier vinden als we hier to_save gebruiken, dat is meer "programmeerderig"

File: coursedata/adventures/nl.yaml [hunk 2]
```diff
@@ -1547,6 +1620,29 @@ adventures:
                         print ' Hoera! Je bent ontsnapt!'
                     ```
                 start_code: "print 'Ontsnap uit het spookhuis!'"
+    piggybank:
+            name: Spaarvarken
+            description: Tel je zakgeld!
+            image: hauntedhouse.png
```
Comment (Felienne) [inline coursedata/adventures/nl.yaml:1626]: Hebben we hier nog een Felienne-original nodig? Maak maar een issue zodat ik het niet vergeet :)

**Review by Felienne — CHANGES_REQUESTED** [review]

File: coursedata/adventures/nl.yaml (hunk 2 above)
Comment (MarleenGilsing) [reply coursedata/adventures/nl.yaml:1626]: Check!

File: coursedata/adventures/en.yaml (hunk 1 above)
Comment (MarleenGilsing) [reply coursedata/adventures/en.yaml]: Done

**Pull request closed without merging**